frontend/components/Auction.vue

```diff
@@ -233,12 +233,31 @@
                 </div>
             </TextBlock>
         </div>
-        <Loading v-else-if="isAuctionsLoading" is-loading class="w-full self-center Loading h-48" />
+        <div v-else-if="takeEvents" class="mt-5">
+            <p class="mb-2">
+                The auction was taken via {{ takeEvents.length }} transaction<span v-if="takeEvents.length !== 1"
+                    >s</span
+                >
+                at {{ takeEvents[takeEvents.length - 1].date.toISOString() }}.
+            </p>
+            <ul class="list-disc list-inside">
+                <li v-for="(event, index) of takeEvents" :key="index">
+                    Transaction <FormatAddress :value="event.transactionHash" :shorten="true" /> (<TimeTill
+                        :date="event.date"
+                    />)
```

Comment:
I would imagine it can be something like `Transaction 0x000..000 executed 2h ago`
Also, since `transactionDate` can be undefined, I suppose this should be handled somehow